scaffold/Taskfile.yml

```diff
@@ -14,6 +14,11 @@ includes:
 output: prefixed
 silent: true
 
+# Only allow tasks to be run once per task invocation. This ensures that
+# multiple dependencies on the same task are only run once.
+# @see https://github.com/go-task/task/issues/812
+run: once
```

Comment:
So does this pass down to included task files as well? Or just tasks declared here directly?

Reply:
I am not sure. The official documentation is at https://taskfile.dev/usage/#limiting-when-tasks-run (maybe we should reference that rather than the GitHub issue). I'm expecting it only works within the current file, considering [variables need to be manually passed to includes](https://taskfile.dev/usage/#vars-of-included-taskfiles).